README.md

```diff
@@ -28,10 +28,16 @@ You can find all the documentation for LunarVim at [lunarvim.org](https://www.lu
 
 Make sure you have the release version of Neovim (0.6).
 
+Linux:
```

Comment:
maybe `Linux & Mac` or even `Unix Family` 🤔

Reply:
I agree that "Linux" doesn't encompass enough. As a primarily Windows user, I'm not too familiar with what Linux/OSX people look for with these kinds of things, so I will defer to someone else.

Reply:
i mean we can always tune it later on, tyvm for your contribution,
@kylo252 any thoughts on this?